source/index.txt

```diff
@@ -52,10 +52,10 @@ Get Started
 .. toctree::
    :titlesonly:
 
-   /install
-   /connect
-   /databases-collections
-   /playgrounds
-   /create-cluster-terraform
-   /reference
+   Install </install>
+   Connect </connect>
+   Manage Your Data </databases-collections>
```

Comment:
(suggestion)
You could use "Manage Data" for further consolidation, if you'd like?